packages/sdk-components-react/src/vimeo.tsx

```diff
@@ -68,7 +62,7 @@ type VimeoPlayerOptions = {
   transparent?: boolean;
 };
 
-const getUrl = (options: VimeoPlayerOptions) => {
+const getVideoUrl = (options: VimeoOptions) => {
```

Comment:
Looks like a bug
In UI I see enter video URL, Im entering `https://player.vimeo.com/video/831343124`

The code below transforms it into

```
    const userUrl = new URL(options.url);
    url = new URL(IFRAME_CDN);
    url.pathname = `/video${userUrl.pathname}`;
```

causing double `/video/video` to appear

Reply:
In the UI the expection is to enter vimeo url that you get form the vimeo UI 
<img width="320" alt="image" src="https://github.com/webstudio-is/webstudio/assets/52824/ba864f29-b766-4faf-a6b4-6f678452915c">